tests/jax/core_system_components/config_test.py

```diff
@@ -70,14 +132,40 @@ def config() -> Config:
     return Config()
 
 
-def test_add_single_config(config: Config, dummy_component_config: type) -> None:
+@pytest.fixture
+def dummy_component_without_config_class() -> Component:
+    """Creates a component with no config class.
+
+    Returns:
+        instantiation of a Mava Component
+    """
+
+    dummy_component = DummyComponentNoConfigClass(config=SimpleNamespace(param_1=1))
+
+    return dummy_component
```

Comment:
just return no need to store the variable

Reply:
Sorted.